src/locales/en.json

```
@@ -377,5 +377,10 @@
     "more": "More",
     "exposure_history": "Exposure History",
     "more_info": "More Info"
+  },
+  "errors": {
+    "title": "Whoops!",
```

```suggestion
    "title": "Unknown Error",
```

I understand that this still needs to go through design and stuff, but its possible to see this during a demo or something before that happens and given this is an app for health related issues a more formal tone is more appropriate here.